### PR TITLE
Added DB connection

### LINKED ISSUE #30: Add DB connection
Add dependencies to `pom.xml` and a test servlet that connects to the databases and shows the result of a test query.

### PR DESCRIPTION
**Pull request by San7o**
Added db connection an the api /TestDB. This closes #30 

Here is what you need to recreate a test environment:

Run the DB inside docker:
```bash
root@6543ebf53605:/# java -jar /opt/db-derby-10.16.1.1-bin/lib/derbyrun.jar server start&
```

Add a table with some content (for the test, the table must be called "users" and it has to have a "name" column)
```bash
root@6543ebf53605:/# java -jar /opt/db-derby-10.16.1.1-bin/lib/derbyrun.jar ij
ij> CONNECT 'jdbc:derby://localhost:1527/DemoDB;create=true';
ij> CREATE TABLE users(ID INT PRIMARY KEY, Name VARCHAR(12));
ij> INSERT INTO users VALUES(1, 'Marco'),(2, 'Enricooo');
ij> exit;
```
Now when you connect to the DB, the page should return the values inserted into users

### CODE REVIEW

File: src/main/java/TestDB.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  {
+    
+    try {
+
+        // Loading the driver
+        Class.forName("org.apache.derby.jdbc.ClientDriver");
+    }
+    catch(ClassNotFoundException e) {
+
+        PrintWriter out = res.getWriter();
```
Comment (Qw3rty26) [inline src/main/java/TestDB.java]: Line 27, together with line 42 and 54 can be put once at the start of the doGET method.

**Review by Qw3rty26 — CHANGES_REQUESTED** [review]
@San7o everything looks fine, just do a little refactor and we should be good to go.

Try to put this line:

`PrintWriter out = res.getWriter();`

only once in the `TestDB.java` servlet.


Also, this servlet isn't printing a page by using the "include" method as discussed in #6. I guess printing variables, errors and such is difficult while using the "include" method. Maybe there's a workaround? If not then printing the error without "include", like you are doing here, should be fine as well.

**Comment by San7o** [comment]
The error page is still non functional, I cannot display the error in /error yet... So I'm using the writer to print the error (since  we need this api for debugging purposes).
I refactored the code, but i think the line `PrintWriter out = res.getWriter();` should be initialized right before using it. This helps with readability, since you don't need to look up at the start of the file (and for very big files, this is best practice). It also helps with maintainability: if I need to change the error display, I will only modify the code inside the catch statement without touching any other code, leaving everything logically separated.

**Review by Qw3rty26 — APPROVED** [review]
lgtm

**Pull request merged**